python/tvm/relay/qnn/op/legalizations.py

```diff
@@ -72,6 +72,7 @@ def legalize_qnn_unary_op(attrs, inputs, types):
 register_qnn_unary_op_legalize("qnn.erf", special.erf)
 register_qnn_unary_op_legalize("qnn.sigmoid", lambda arr: 1 / (1 + np.exp(-arr)))
 register_qnn_unary_op_legalize("qnn.tanh", np.tanh)
+register_qnn_unary_op_legalize("qnn.log", np.log)
```

Comment:
@AndrewZhaoLuo Doesn't this already do the lookup table?

Reply:
oh yeah im dumb